__tests__/integration/mirador/basic.test.js

```diff
@@ -11,6 +11,7 @@ describe('Basic end to end Mirador', () => {
     await expect(page).toClick('.mirador-add-resource-button');
     await expect(page).toFill('#manifestURL', 'http://localhost:5000/api/sn904cj3439');
     await expect(page).toClick('#fetchBtn');
+    await expect(page).toClick('button[aria-label="Close Menu"]'); // Close menu so new item is visible
```

Comment:
I believe this will be solved when dealing w/ #2038, but for now, it seemed reasonable to just close the menu so the item is visible in the test.